### PR TITLE
Add improvements to document template selector

### PR DESCRIPTION
**Pull request by Remi749**
### Your checklist for this pull request

- [X] Make sure you are requesting to **pull a topic/feature/bugfix branch** (right side). Don't request your main!
- [X] Make sure you are making a pull request against the **dev** branch (left side). Also you should start *your branch* off *dev*.
- [X] Check the commit's or even all commits' message 
- [X] Check if your code additions will fail linting checks
- [X] Remember: Add issue description to [CHANGELOG](https://github.com/Puzzlepart/prosjektportalen365/blob/dev/CHANGELOG.md) with the **ID of the issue** associated with this PR
- [X] Documentation: Have a look at the [PP365 User manual](https://puzzlepart.github.io/prosjektportalen-manual/) and consider the need for updates to be made. Updates can be done directly into the 'Kladd' branch or by providing information to test team for implementation.

#### Improvements:
- Sort 'Malbibliotek' view the same way as the 'Malbibliotek' library -> By name
- Adjust max-height for the dialog to 90%
  - To hinder the dialog to go out of the windows view and instead add breathable space to bottom and top (this is for 'Malbiblotek' that has a lot of documents in it)
- TargetFolderScreen
  - Sort TargetFolderScreen folders by name
  - Remove possibility to select multiple as this did nothing
- Update information messages to reflect the TargetFolderScreen as the first screen doesn't say anything about selecting a target folder

### How to test

1. Open Document Template Selector and see changes metioned above

### Relevant issues (if applicable)

#711

### CODE REVIEW

File: SharePointFramework/ProjectExtensions/src/components/DocumentTemplateDialog/TargetFolderScreen/index.tsx [hunk 1]
```diff
@@ -54,9 +54,9 @@ export const TargetFolderScreen = () => {
         setFolder={setFolder}
       />
       <DetailsList
-        items={folders}
+        items={folders.sort((a, b) => (a.name > b.name ? 1 : -1)) }
         columns={columns()}
-        selectionMode={SelectionMode.multiple}
+        selectionMode={SelectionMode.none}
```
Comment (olemp) [inline SharePointFramework/ProjectExtensions/src/components/DocumentTemplateDialog/TargetFolderScreen/index.tsx:59]: We don't need selection here?

File: SharePointFramework/ProjectExtensions/src/components/DocumentTemplateDialog/SelectScreen/index.tsx [hunk 2]
```diff
@@ -25,7 +25,7 @@ export const SelectScreen = (props: ISelectScreenProps) => {
       context.templates
         .filter((item) => {
           return !isEmpty(folder) ? folder === item.parentFolderUrl : item.level === 1
-        })
+        }).sort((a, b) => (a.name > b.name ? 1 : -1))
         .sort((a, b) => (a.isFolder === b.isFolder ? 0 : a.isFolder ? -1 : 1)),
```
Comment (olemp) [inline SharePointFramework/ProjectExtensions/src/components/DocumentTemplateDialog/SelectScreen/index.tsx:29]: Folders will still come first?

**Review by olemp — CHANGES_REQUESTED** [review]

File: SharePointFramework/ProjectExtensions/src/components/DocumentTemplateDialog/SelectScreen/index.tsx (hunk 2 above)
Comment (Remi749) [reply SharePointFramework/ProjectExtensions/src/components/DocumentTemplateDialog/SelectScreen/index.tsx:29]: Yes ;)

File: SharePointFramework/ProjectExtensions/src/components/DocumentTemplateDialog/TargetFolderScreen/index.tsx (hunk 1 above)
Comment (Remi749) [reply SharePointFramework/ProjectExtensions/src/components/DocumentTemplateDialog/TargetFolderScreen/index.tsx:59]: At least as far as I can see, multiple selection makes noe sense here as it doesn't work selecting multiple folders for targets. If you know why multiple selection was set, please explain :)

**Comment by olemp** [comment]
Resolve CHANGELOG conflict @Remi749 

image

**Comment by Remi749** [comment]
> Resolve CHANGELOG conflict @Remi749
> 
> image

done ✔

**Review by olemp — APPROVED** [review]
TGIF

**Pull request merged**